### PR TITLE
Increase Test Coverage

### PR DESCRIPTION
**Pull request by koloz193**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: l1-contracts/contracts/bridgehub/MessageRoot.sol [hunk 1]
```diff
@@ -157,15 +157,15 @@ contract MessageRoot is IMessageRoot, ReentrancyGuard {
...
-    function _unsafeResetChainRoot(uint256 _index, bool _updateTree) internal {
-        uint256 chainId = chainIndexToId[_index];
-        bytes32 initialRoot = chainTree[chainId].setup(CHAIN_TREE_EMPTY_ENTRY_HASH);
-
-        if (_updateTree) {
-            // slither-disable-next-line unused-return
-            sharedTree.updateLeaf(_index, Messaging.chainIdLeafHash(initialRoot, chainId));
-        }
-    }
+    // function _unsafeResetChainRoot(uint256 _index, bool _updateTree) internal {
```
Comment (StanislavBreadless) [inline l1-contracts/contracts/bridgehub/MessageRoot.sol:160]: ?

Comment (koloz193) [reply l1-contracts/contracts/bridgehub/MessageRoot.sol:160]: its not used anywhere yet and i was receiving some linting errors

File: l1-contracts/contracts/bridgehub/MessageRoot.sol [hunk 2]
```diff
@@ -154,46 +154,4 @@ contract MessageRoot is IMessageRoot, ReentrancyGuard {
         // slither-disable-next-line unused-return
         sharedTree.updateAllLeaves(newLeaves);
     }
-
-    // It is expected that the root is present
-    // `_updateTree` should be false only if the caller ensures that it is followed by updating the entire tree.
-    function _unsafeResetChainRoot(uint256 _index, bool _updateTree) internal {
```
Comment (kelemeno) [inline l1-contracts/contracts/bridgehub/MessageRoot.sol:160]: This is deleted because it is not used for now, right?

Comment (koloz193) [reply l1-contracts/contracts/bridgehub/MessageRoot.sol:160]: yup and the future of the calling method is in the air

**Review by kelemeno — APPROVED** [review]

**Pull request merged**